lib/message.rb

```diff
@@ -0,0 +1,30 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+class Message
+
+  def initialize
+    @client = Twilio::REST::Client.new ENV['ACCOUNT_SID'], ENV['AUTH_TOKEN']
+  end
+
+  def send_text(to)
+    @client.messages.create(
```

Comment:
Could this formatting be a little tidier? Have a look at a Ruby style guide for guidance on calling a method with a lot of args.